index.html

```diff
@@ -161,15 +206,18 @@ <h1>
         <h2>
 Target system
         </h2>
-        <p class="issue" title="DIDs in DNS">
-Explain the difference from / relationship with the Decentralized Identifier
-in DNS [[?DID-DNS]] spec.
-        </p>
+        
         <p>
 The target system of the Web DID method is the web host that the domain name
 described by the DID resolves to when queried through the Domain Name System
 (DNS).
         </p>
+        
+        <p>
+          Unlike [[?DID-DNS]], where the DID is identified as part of a DNS URI 
+          Resource Record, did:web permits both discovery of and delivery of a 
```

Comment:
I think we don't need to comment on DID DNS.

Reply:
agreed - not necessary,  but not harmful (and potentially helpful to folks that are net new to dids [pun intended]) and is a question i have heard asked by multiple folks many times

Reply:
IMO, its harmful to connect reader to DID DNS... i'm a -1 to including references to it.

Reply:
-1 to including references to `did:dns`, +1 to removing.